.github/workflows/ci.yml

```
@@ -716,7 +716,7 @@ jobs:
       - name: "Generate back references for helm-chart"
         run: breeze release-management add-back-references helm-chart
       - name: Configure AWS credentials
-        uses: ./.github/actions/configure-aws-credentials
+        uses: aws-actions/configure-aws-credentials@010d0da01d0b5a38af31e9c3470dbfdabdecca3a  # v4.1.0
```

```suggestion
        uses: aws-actions/configure-aws-credentials@010d0da01d0b5a38af31e9c3470dbfdabdecca3a  # v4.0.1
```
And I am guessing 
`uses: aws-actions/configure-aws-credentials@v4.0.1` could work directly too 🤔

---

We should avoid using tags for security reasons, because they can recreate the tag with different commit

---

> We should avoid using tags for security reasons, because they can recreate the tag with different commit

Precistely - see also my commit message explaining  that :)

---

We only use tags for Actions that are directly provider by GitHub (this is also why we used submodules before - submodules provide the same kind of functionality - they link the external repo in specific commit - commit hash is stored in git repo as reference and the action checked out via submodule has always been checked in the version that submodule was updated to. 

But over time I found submodules are still not well understood by people and likely there is a good reason for that. You probably would not even know how to switch to a newer version of the action with submodules. Again - that's the n-th time when I think submodules are cool for something and I add them but at the moment it requires "others" to understand and use them and regularly update to newer versions of linked repositories, submodules are simply super-confusing, not well understood and generally broken.

I'd only recommend submodules if you have an "admin  team" managing huge set of linked repos, where you do not require "regular" contributors or even maintainers of each of the repos to do anything with these.